components/size_aware_image.jsx

```diff
@@ -170,6 +196,79 @@ export default class SizeAwareImage extends React.PureComponent {
             />
         );
 
+        // copyLink, download are two buttons overlayed on image preview
+        // copyLinkTooltip, downloadTooltip are tooltips for the buttons respectively.
+        // if linkCopiedRecently is true, defaultMessage would be 'Copy Link', else 'Copied!'
+
+        const copyLinkTooltip = (
+            <Tooltip
+                id='copy-link-tooltip'
+                className='hidden-xs'
+            >
+                <FormattedMessage
+                    id='single_image_view.copy_link_tooltip'
+                    defaultMessage={this.state.linkCopiedRecently ? 'Copied' : 'Copy link'}
+                />
+            </Tooltip>
+        );
+        const copyLink = (
+            <OverlayTrigger
+                className='hidden-xs'
+                delayShow={500}
+                placement='top'
+                overlay={copyLinkTooltip}
+                rootClose={true}
+            >
+                <button
+                    className={classNames('style--none', 'size-aware-image__copy_link', {
+                        'size-aware-image__copy_link--recently_copied': this.state.linkCopiedRecently,
+                    })}
+                    aria-label={`${ariaLabelImage}${localizeMessage('copy url')}`}
```

Comment:
This actually has to be done a bit differently. We can't  concatenate translated strings like this since some languages may structure text differently. That said, I think we don't need to include the image name in this label either since we'd just be repeating ourselves, so I think we can just have this aria label be something like "Copy link" without the rest of it.

Also, `localizeMessage` works like `FormattedMessage` where you need to pass in two arguments, an ID and a default message, so it would need to be called as `localizeMessage('single_image_view.copy_link_tooltip', 'Copy link')`